### PR TITLE
UI Revamp

### PR DESCRIPTION
**Pull request by pedroseac**
VueJS on all pages, and UI Revamp.

* Added support for VueJS and refactored pages using it
* Separated files around, now each page has is own js controller file
* Created component for navigation bar in VueJS
* Revamped UI using bulma.io (Bye Bootstrap!)
* Changed datepicker from events page

WARNING: I moved all source files into an src folder, careful when merging

TODO: use Vue routing system for SPA

---
This change is [Reviewable](https://reviewable.io/reviews/sharifmarat/vb/5)

### CODE REVIEW

File: cgi-bin/init.sh [hunk 1]
```diff
@@ -10,5 +10,5 @@ fi
 VOLLEY_DB=$DB VOLLEY_DEBUG=1 QUERY_STRING=action=init ./vb.cgi
 
 # temporary add an event and a guest for testing
-VOLLEY_DB=$DB VOLLEY_DEBUG=1 QUERY_STRING=action=add_event\&date=test_date\&location=location\&payment_link=tbd ./vb.cgi
+VOLLEY_DB=$DB VOLLEY_DEBUG=1 QUERY_STRING=action=add_event\&date="11/07/2018 19:30"\&location=Pijp\&payment_link=tbd ./vb.cgi
```
Comment (wluberti) [inline cgi-bin/init.sh]: Isn't the location BlauwWit?

File: src/js/index.js [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        });
+      }
+    }
+  })
+
+  const currentEventInfo = new Vue({
+    el: '#current-event-info',
+    template: `
+      <p>
+        Most recent event:
```
Comment (sharifmarat) [inline src/js/index.js]: `Most recent event` was gone already, by the way

File: src/js/index.js [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+        <strong>
+          <label>{{ start }}</label>
+          <label>{{ location }}</label>
+        </strong>
+        &nbsp;
+        <span v-if="start">
+          <span title="Add to Calendar" class="addeventatc">
```
Comment (sharifmarat) [inline src/js/index.js]: The way how date is displayed changed. Before `Wednesday, Jul 11th 2018 at 19:30`, after `31.07.2018 19:30`.

File: index.html [hunk 4]
```diff
@@ -1,143 +1,54 @@
...
-                <span class="navbar-brand">Let's volleyball</span>
-                <ul class="navbar-nav mr-auto">
-                  <li class="nav-item active">
-                    <a class="nav-link" href="#">Primary event</a>
-                  </li>
-                  <li class="nav-item">
-                    <a class="nav-link" href="events.html">All events</a>
-                  </li>
-                  <li class="nav-item">
-                    <a class="nav-link" href="shame.html">Shame</a>
```
Comment (sharifmarat) [inline index.html]: This change breaks navigation (`Shame` is missing for example).

File: index.html [hunk 5]
```diff
@@ -1,143 +1,54 @@
...
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic|Anonymous+Pro:400,700,400italic,700italic|Merriweather:400,700,300">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" href="src/css/main.css?v=1.0.2">
 
-            $('.addeventatc .timezone').text('Europe/Amsterdam');
-            $('.addeventatc .title').text('Let\'s volleyball @' + result.message.location + ' on ' + m_date.format(display_format));
-            $('.addeventatc .description').text(result.message.date + ' ' + result.message.location);
-            $('.addeventatc .location').text(result.message.location);
+  <!--  js -->
+  <script src="https://cdn.jsdelivr.net/npm/vue@2.5.16/dist/vue.min.js"></script>
```
Comment (sharifmarat) [inline index.html]: Would `vue.min.js` replace jquery?

Comment (pedroseac) [reply index.html]: I'm still using `$.ajax`, so still need that bit. Gonna work to change it to [fetch](https://developers.google.com/web/updates/2015/03/introduction-to-fetch)

File: src/js/index.js (hunk 2 above)
Comment (pedroseac) [reply src/js/index.js]: removed

File: index.html (hunk 4 above)
Comment (pedroseac) [reply index.html]: readded

File: src/js/index.js (hunk 3 above)
Comment (pedroseac) [reply src/js/index.js]: fixed

**Pull request merged**